src/pages/edge/identity-for-edge-network/faq.md

```diff
@@ -113,6 +113,16 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
+* Cleared when the app is uninstalled.
```

Comment:
I think to make this not fail validation, there needs to be a newline between here and the beginning of the unordered list.